### DISCUSSION TITLE
How does the segmenting in Q&A format works?

### DISCUSSION THREAD

**Discussion by dailydaniel** [langgenius/dify#7240] [Help] [answered]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] [FOR CHINESE USERS] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

I use Q&A mode for segmentation in chatflow. I thought that questions are vectorized and searched, and the answers are in the metadata so as not to spoil the search, but in the output of chunks in the chatflow, both the question and the answer are in context. So I want to know how this segmentation works in terms of search - does it go only by questions (which is logical) or by both the question and the answer.

**Comment by patryk20120** [comment, marked as answer]
Chunk contain both question and the answer, when using vector search it will look for similarity in chunk meaning both question and answer
